lib/std/os/windows.zig

```diff
@@ -1923,6 +1923,7 @@ pub const CreateProcessError = error{
     InvalidName,
     NameTooLong,
     InvalidExe,
+    CommitmentLimit,
```

Comment:
Should probably be `SystemResources` for consistency with other functions?

Reply:
I was going to suggest this as well.